gnosis/safe-react
# feat: filter form

## What it solves
Visual implementation of filter mockup

## How this PR fixes it
On the historical transaction list, the filter form has been implemented (sans functionality).

By editing the values and clicking 'Apply', the basic filter will be logged. The data structure will need to be adjusted to match that of backend and filtered of falsy values.

This PR also adds React Hook Form which we intend to migrate to.

## Known bugs
- [x] Sometimes fields do not clear successfully.
- [x] Domain loading indicator is on new line.
- [x] The full address is not shown.
- [x] The entered address book name/domain is not preserved when the filter is re-opened.
- [x] Inputs jumping on input (because of changing width)
- [ ] Text overlap in the module select

## How to test it
Enter values in the filter and click apply, observe the logged filter. The form values should be preserved when re-opening the filter.
